packages/react-ui-ag/src/Filters/BathroomFilterCard.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import { RadioGroup } from '@rentpath/react-ui-core'
+import autobind from 'autobind-decorator'
+import isEqual from 'lodash/isEqual'
+import FilterCard from './FilterCard'
+
+@themed(/^BathroomFilterCard/)
+export default class BathroomFilterCard extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    title: PropTypes.string,
+    description: PropTypes.string,
+    fields: PropTypes.arrayOf(
+      PropTypes.shape({
+        label: PropTypes.string,
```

Comment:
`label` should match the `Field` label type.
```
label: PropTypes.oneOfType([
  PropTypes.node,
  PropTypes.func,
  PropTypes.object,
])
```

Reply:
Good catch - forgot that one! 👍